recipes/civetweb/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2018-2020 the CivetWeb developers
+ * MIT License
+ */
+
+/* Simple demo of a REST callback. */
+#ifdef _WIN32
+#include <windows.h>
+#else
+#include <unistd.h>
+#endif
+
+#include <stdlib.h>
+#include <string.h>
+#include <time.h>
+
+#include "civetweb.h"
+
+#define PORT "8080"
+#define HOST_INFO "http://localhost:8080"
+#define EXAMPLE_URI "/example"
+
+static int exitNow = 0;
+
+static int
+ExampleGET(struct mg_connection* conn)
+{
+    mg_send_http_ok(conn, "text/plain", 10);
+    exitNow = 1;
+	return 200;
+}
+
+static int
+ExampleHandler(struct mg_connection *conn, void *cbdata)
+{
+	const struct mg_request_info *ri = mg_get_request_info(conn);
+	(void)cbdata; /* currently unused */
+
+	if (0 == strcmp(ri->request_method, "GET")) {
+		return ExampleGET(conn);
+	}
+	return 405;
+}
+
+int
+log_message(const struct mg_connection *conn, const char *message)
+{
+	puts(message);
+	return 1;
+}
+
+int
+main(int argc, char *argv[])
+{
+	struct mg_callbacks callbacks;
+	struct mg_context *ctx;
+	time_t start_t;
+	time_t end_t;
+	double diff_t;
+	int err = 0;
+
+	mg_init_library(0);
```

Comment:
The `test_package` is as minimal as possible this one line should be enough to make sure everything is build and links